arches/app/datatypes/url.py

```diff
@@ -255,3 +264,8 @@ def default_es_mapping(self):
                 },
             }
         }
+
+    def pre_tile_save(self, tile, nodeid):
+        if tile.data[nodeid]:
+            if tile and "url_label" not in tile.data[nodeid]:
```

Comment:
Now `if tile` is redundant, as we've already checked something more specific.
```suggestion
            if "url_label" not in tile.data[nodeid]:
```

Probably could just be consolidated, at your option:
```py
    if (tile_val := tile.data[nodeid]) and "url_label" not in tile_val:
        tile_val["url_label"] = ""
```

Reply:
Added in [685a5c6](https://github.com/archesproject/arches/pull/10642/commits/685a5c682bbac3614a4d44a789768440c4f755d4)